kaggle-classification/keras_trainer/model.py

```diff
@@ -115,13 +115,11 @@ def predict(self, texts):
 
   def score_auc(self, data):
     predictions = self.predict(data['comment_text'])
-    scores = []
-    for idx, label in enumerate(LABELS):
-      labels = np.array(data[label])
-      score = metrics.roc_auc_score(labels, predictions[idx].flatten())
-      scores.append(score)
-      print('{} has AUC {}'.format(label, score))
-    print('Avg AUC {}'.format(np.mean(scores)))
+    labels = np.array(list(zip(*[data[label] for label in LABELS])))
+    individual_auc_scores = metrics.roc_auc_score(labels, predictions, average=None)
+    print("Individual AUCs: {}".format(list(zip(LABELS, individual_auc_scores))))
+    mean_auc_score = metrics.roc_auc_score(labels, predictions, average='macro')
+    print("Mean AUC: {}".format(mean_auc_score))
```

Comment:
Don't change it in this PR, but we should switch from `print` to using the TensorFlow logging library, `tf.logging()`